css/css-writing-modes-3/table-cell-001.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html lang="en">
+<meta charset="utf-8">
+<title>CSS Writing Mode test: orthogonal table cell</title>
+<link rel="help" href="https://drafts.csswg.org/css-writing-modes-3/#block-flow">
+<meta assert="The writing mode property applies to table cells: two adjacent cells can have orthogonal writing modes, and the enclosing table must be laid out correctly.">
+<meta flags="ahem">
+<link rel="match" href="reference/table-cell-001-ref.html">
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+
+<style>
+@font-face {
```

Comment:
I usually don't include it, and then people complain that the test looks weird when they look at it on a computer they didn't set up properly, such as edge in browser stack, so I decided to play it safe and include the font.

Also, there are a bunch of version of ahem around, and it seems sane to require one that is known to work with the test (even though this particular test should work with any ahem).

Should I remove it?